pages/docs/reference/functions/create.mdx

```diff
@@ -159,6 +159,13 @@ The `createFunction` method accepts a series of arguments to define your functio
 
 One of the following function triggers is **Required**.
 
+You can also specify an array of up to 10 of the following triggers to trigger your function
```

Comment:
Would it be okay to say "to invoke your function" to avoid repetition?

Reply:
Much better! Thank you ❤️